server/eclipse-project/src/main/java/us/freeandfair/corla/Main.java

```diff
@@ -4,7 +4,6 @@
  * @title ColoradoRLA
  * @created Jul 19, 2017
  * @copyright 2017 Colorado Department of State
- * @license GNU Affero General Public License v3 with Classpath Exception
  * @creator Daniel M. Zimmerman <dmz@freeandfair.us>
```

Comment:
I think that as Chris suggests we should follow FSF advice:

https://www.gnu.org/licenses/gpl-howto.html
> Put a license notice in each file.

I'd substitute this instead:
> @license SPDX-License-Identifier: AGPL-3.0-only
or this
> @license SPDX-License-Identifier: AGPL-3.0-or-later

Reply:
I agree.  I will make this update now.

Reply:
You are welcome to review again, @nealmcb.  The push is making its way through CI.